<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsMenu</class>
 <widget class="QWidget" name="SettingsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Настройки сервера</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  
  <widget class="QWidget" name="ipLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>460</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ipLayout">
    <item>
     <widget class="QLabel" name="ip_label">
      <property name="text">
       <string>IP-адрес сервера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip_input">
      <property name="text">
       <string>localmachine</string>
      </property>
      <property name="placeholderText">
       <string>Введите IP-адрес</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  
  <widget class="QWidget" name="portLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>460</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="portLayout">
    <item>
     <widget class="QLabel" name="port_label">
      <property name="text">
       <string>Порт сервера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port_input">
      <property name="text">
       <string>1313</string>
      </property>
      <property name="placeholderText">
       <string>Введите порт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  
  <widget class="QWidget" name="buttonLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>460</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="save_btn">
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="default_btn">
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>По умолчанию</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>265</y>
     <width>460</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  
 </widget>
 <resources/>
 <connections/>
</ui> 